external/avro/src/test/resources/messy.avsc

```diff
@@ -0,0 +1,140 @@
+{
```

Comment:
Instead of adding this complicated avro json schema, should we test smaller cases as unit tests in `AvroCatalystDataConversionSuite` so in the future, it will be easier to know what is broken? Thanks.